Dockerfile

```
@@ -0,0 +1,14 @@
+FROM node:latest
+WORKDIR /code
+COPY . /code
+
+ARG baseURL
+ENV BASE_URL ${BASE_URL}
```

not sure if this would be the reason for the deploy not working, but would definitely cause problems later

```suggestion
ENV BASE_URL ${baseURL}
```